react-common/components/theming/themeManager.ts

```diff
@@ -30,6 +30,10 @@ export class ThemeManager {
         return this.currentTheme;
     }
 
+    public isKnownTheme(themeId: string): boolean {
+        return pxt.appTarget.colorThemeMap && themeId in pxt.appTarget.colorThemeMap;
```

Comment:
nitpick: we typically avoid the `in` operator because it has some weird semantics. probably fine here, but consider changing for consistency